cmd/chroma/main.go

```diff
@@ -21,11 +21,11 @@ import (
 	colorable "github.com/mattn/go-colorable"
 	isatty "github.com/mattn/go-isatty"
 
-	"github.com/alecthomas/chroma/v2"
-	"github.com/alecthomas/chroma/v2/formatters"
-	"github.com/alecthomas/chroma/v2/formatters/html"
-	"github.com/alecthomas/chroma/v2/lexers"
-	"github.com/alecthomas/chroma/v2/styles"
+	"github.com/TempleOS-Simplified/chroma/v2"
```

Comment:
This can't be merged.

Reply:
That was a test. And that wasn't commited in the pull request I don't think.

Reply:
Edit: It was, apologies. I added that for testing, it's removed now.